devel/apache-arrow/Portfile

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        apache arrow 0.17.1 apache-arrow-
+revision            0
+name                ${github.author}-${github.project}
+
+platforms           darwin
+license             Apache-2
+maintainers         nomaintainer
+
+description         Powering In-Memory Analytics
+
+long_description    Apache Arrow is a development platform for in-memory\
+                    analytics. It contains a set of technologies that\
+                    enable big data systems to process and move data\
+                    fast.
+
+homepage            https://github.com/apache/arrow
+distname            ${github.author}-${github.project}-${version}
+
+# Build instructions: https://github.com/apache/arrow/blob/master/docs/source/developers/python.rst#build-and-test
+set build_dir       ${worksrcpath}/cpp/build
+post-extract {
+    xinstall -d ${build_dir}
+}
+
+# require c++14
+compiler.cxx_standard   2014
+
+universal_variant   no
+
+cmake.source_dir    ..
+cmake.build_type    Release
+
+configure.pre_args-append \
+                    -DARROW_FLIGHT=ON \
+                    -DARROW_JEMALLOC=OFF \
+                    -DARROW_ORC=ON \
+                    -DARROW_PARQUET=ON \
+                    -DARROW_PLASMA=ON \
+                    -DARROW_PROTOBUF_USE_SHARED=ON \
+                    -DARROW_PYTHON=ON \
+                    -DARROW_WITH_BZ2=ON \
+                    -DARROW_WITH_ZLIB=ON \
+                    -DARROW_WITH_ZSTD=ON \
+                    -DARROW_WITH_LZ4=ON \
+                    -DARROW_WITH_SNAPPY=ON \
+                    -DARROW_WITH_BROTLI=ON \
+                    -DARROW_INSTALL_NAME_RPATH=OFF
+    
+# create Python subports
+set python_versions {37 38}
+foreach v ${python_versions} {
+    subport py${v}-py${github.project} {
+        PortGroup   python 1.0
+        python.default_version  ${v}
+    }
+}
+
+if {${name} eq ${subport}} {
+    checksums       rmd160  7c42bb243b87a9ddb8458c43f240fd8d117e40cb \
+                    sha256  b7e0020dd31ae126500ae865248c01bcfc6dbeefbbe223ed0b95339c5f0932c7 \
+                    size    5835020
+
+    # Apache Arrow requires a python3 version to build; use this
+    set python_branch   3.7
```

Comment:
Please use 3.8; it's the latest stable version.

Reply:
Thanks done.